stacks-signer/Cargo.toml

```diff
@@ -46,6 +46,8 @@ url = "2.1.0"
 
 [dev-dependencies]
 clarity = { path = "../clarity", features = ["testing"] }
+polynomial = "0.2.6"
+num-traits = "0.2.18"
```

Comment:
Where do these get used?

Reply:
In the SignerDB tests (line 212 and 213 on the current commit).